src/common/Navigation.js

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+
+function Navigation(props) {
```

Comment:
This name is not specific enough. What kind of navigation is this? Is it a nav bar or is it a side menu. Can you just be a bit more specific?

Reply:
I have changed the name to ReusableNavLink. It is for the nav links rather than the navigation itself. For example, login, signup. I don't know if that makes sense.